### PR TITLE
v5.33 Notice

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by esethna — APPROVED** [review]

File: notices.json [hunk 1]
```diff
@@ -37,21 +37,21 @@
...
+      "serverVersion": ["<5.33"],
       "instanceType": "onprem",
-      "displayDate": ">= 2021-02-21T00:00:00Z"
+      "displayDate": ">= 2021-03-21T00:00:00Z"
     },
     "localizedMessages": {
       "en": {
         "title": "New server version available",
-        "description": "The new feature release offers a streamlined license renewal experience, updated incident status and task list, and custom, collapsible channel categories. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.",
+        "description": "The new feature release offers OpenID Connect for OAuth 2.0 authentication, incident collaboration timeline, and custom Mattermost status. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.",
```
Comment (cwarnermm) [inline notices.json:51]:
```suggestion
        "description": "The new feature release offers OpenID Connect for OAuth 2.0 authentication, incident collaboration timeline, and custom Mattermost user statuses. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.",
```

**Review by cwarnermm — COMMENTED** [review]
Proposed update provided inline.

**Comment by cwarnermm** [comment]
@esethna - I adjusted the language around custom statuses for this notice. Any concerns from your end?

**Comment by esethna** [comment]
@cwarnermm perhaps we drop the word "mattermost"?

File: notices.json [hunk 2]
```diff
@@ -37,21 +37,21 @@
...
+      "serverVersion": ["<5.33"],
       "instanceType": "onprem",
-      "displayDate": ">= 2021-02-21T00:00:00Z"
+      "displayDate": ">= 2021-03-21T00:00:00Z"
     },
     "localizedMessages": {
       "en": {
         "title": "New server version available",
-        "description": "The new feature release offers a streamlined license renewal experience, updated incident status and task list, and custom, collapsible channel categories. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.",
+        "description": "The new feature release offers OpenID Connect for OAuth 2.0 authentication, incident collaboration timeline, and custom Mattermost user statuses. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.",
```
Comment (cwarnermm) [inline notices.json:51]:
```suggestion
        "description": "The new feature release offers OpenID Connect for OAuth 2.0 authentication, incident collaboration timeline, and custom user statuses. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.",
```

**Review by cwarnermm — COMMENTED** [review]
Small update to wording based on PM feedback

**Review by esethna — APPROVED** [review]

**Review by ogi-m — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**